src/utils.jl

```diff
@@ -213,8 +214,17 @@ end
     end
 end
 
-function rank_vec(ig::MetaGraph)
```

Comment:
Why is this removed?

Reply:
How come all tests pass if this function is removed?

Reply:
It is in `ising.jl`